index.html

```diff
@@ -2596,13 +2596,15 @@ <h3>The Principle of Minimum Disclosure</h3>
 attribute that appears on a driver's license in addition to atomized
 credentials (a credential containing only the person's birthday), and atomized
 credentials that are more abstract (a credential containing only an
-<code>ageOver</code> attribute). In addition, the issuer is encouraged to
+<code>ageOver</code> attribute). One possible adaptation is for the issuer to
 provide secure HTTP endpoints for retrieving single-use bearer credentials to
-promote the pseudonymous usage of credentials when it is safe for the issuer to
-issue such credentials.
+promote the pseudonymous usage of credentials. Besides being impractical or unsafe
+for issuers to support this behavior in some cases, this may be less valuable than
+it first appears, because it creates a dependence on issuers at proving time,
+and introduces temporal correlation risk from the issuer.
```

Comment:
I think the way that this is worded makes it seem like the recommendation is pointless. I think worded this way makes it more clear that there are choices (and implies the reasons for making them):

`One possible adaptation` => `One possible simple adaptation`

and

`Besides being impractical... from the issuer` => `Implementers that find this impractical or unsafe should consider the use of selective disclosure schemes that eliminate dependence on issuers at proving time and reduce temporal correlation risk from the issuer.`